packages/server/graphql/types/Company.ts

```diff
@@ -24,9 +25,29 @@ const Company = new GraphQLObjectType<any, GQLContext>({
       resolve: async ({id: domain}, _args: unknown, {dataLoader}) => {
         const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
```

Comment:
> my default org that's created when a user creates an account
Can you look at your local RethinkDB and see what's the `activeDomain` assigned to that org?

The logic to compute `activeDomain` is not simply just looking at the tld. See [here](https://github.com/ParabolInc/parabol/blob/0087c5fda800cd3f5837834c03929b1f70cce689/packages/server/utils/getActiveDomainForOrgId.ts#L1). So it really depends on how your local DB look like and order of users sign-up.

Reply:
The `activeDomain` for my Parabol account is `test.com`, which is the domain I often use when creating dummy test accounts. I can see that the `test.com` domain does correctly count the orgs. I'm not sure why my Parabol account has `test.com` as the active domain instead of `parabol.co`, but it's a separate issue unrelated to this PR that I'll look into.